### PR TITLE
Update build images

### PR DESCRIPTION
**Pull request by albertvaka**
### What does this PR do?

Update build images.

### Motivation

Bump the Ruby version we use to run omnibus.

### CODE REVIEW

**Review by mx-psi — APPROVED** [review]

**Review by KSerrania — COMMENTED** [review]
Looks good to me, but let's wait until #9074 is merged (as the images from this PR would also bump Go to 1.16)

**Pull request merged**
